From 1da75bc529f4df970e53b6c9c5beba0609d8cf16 Mon Sep 17 00:00:00 2001
From: Dmitry Artamonov <dartamonov@dev.rtsoft.ru>
Date: Mon, 4 Jun 2012 18:31:40 +0400
Subject: [PATCH 070/127] video: set 'connected' flag based on 'enabled' state
 of tegra dc

Needed for declaring LVDS-1 as not connected 1st output and
HDMI-1 as second output

Signed-off-by: Dmitry Artamonov <dartamonov@dev.rtsoft.ru>
Signed-off-by: Valentine Barshak <vbarshak@dev.rtsoft.ru>
---
 drivers/video/tegra/dc/dc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/tegra/dc.c b/drivers/gpu/drm/tegra/dc.c
index bfd916e..dee6983 100644
--- a/drivers/gpu/drm/tegra/dc.c
+++ b/drivers/gpu/drm/tegra/dc.c
@@ -1880,7 +1880,7 @@ static int tegra_dc_probe(struct nvhost_device *ndev,
 	if (dc->out_ops && dc->out_ops->detect)
 		dc->out_ops->detect(dc);
 	else
-		dc->connected = true;
+		dc->connected = dc->enabled;
 
 	tegra_dc_create_sysfs(&dc->ndev->dev);
 
-- 
1.7.4.4

